src/rust/engine/src/externs.rs

```diff
@@ -478,6 +476,7 @@ pub enum PyGeneratorResponseType {
 pub struct PyGeneratorResponse {
   res_type: PyGeneratorResponseType,
   values: HandleBuffer,
+  identities: IdentBuffer,
```

Comment:
Add a comment that explains that we store the identity here to force callers to check their value is hashable before returning it?

Reply:
Yep.

Reply:
Fixed this in a commit that closes out one other TODO in this file!